src/commands/scanner/run.ts

```diff
@@ -99,8 +99,22 @@ export default class Run extends ScannerCommand {
 			char: 'v',
 			description: messages.getMessage('flags.vceDescription'),
 			longDescription: messages.getMessage('flags.vceDescriptionLong'),
+			exclusive: ['json'],
+			deprecated: {
+				messageOverride: messages.getMessage('flags.vceParamDeprecationWarning')
+			}
+		}),
+		'severity-for-error': flags.number({
```

Comment:
It feels like there's a better name for this, but I'm not positive what that would be. `minimum-severity`, perhaps?